### DISCUSSION TITLE
Recipe in Beta

### DISCUSSION THREAD

**Discussion by Glycode** [negue/meme-box#475] [Q&A]
Not sure if I was just unable to make it work but if it wasnt me I see an annoying flaw in the recipe function. Previously with the Meta function you could just choose a tag and that would play all of the media with that tag assigned.

Now tho in the recipe when adding the action, you cant add all videos at once when you choose the tag. the tag is just to help find the videos in the filter but you have to assign every video separately as action inside the recipe.

Am I missing something?

**Comment by negue** [comment]
No you are right- now the way is a bit harder to add multiple items.

I'll need to add such an ability again to the recipe selection - so that you can a) select all from a tag and/or b) choose multiple actions at once - but yes so far with the current beta its sadly not possible

sorry for the inconvenience I'll try to add such a feature as soon as possible - yeah its sadly not that great yet - thats why this information right now is quite valueable for the full release that'll be created in the next month(ish or 2)

**Comment by negue** [comment]
I've started the functionality to select a tag when you are in the Action List Dialog (for trigger random) - dunno when its ready but I'll respond here once it is

**Reply by negue** [reply]
![image](https://user-images.githubusercontent.com/842273/184199294-e023689f-d2b2-4154-8845-b86ae76ebbf6.png)

Preview of the Steps atleast improved but the Setting of that Step is currently what takes time - also in that heat here it takes longer haha

**Comment by negue** [comment]
@Glycode - got a beta-2 update - see https://github.com/negue/meme-box/releases/tag/2022.1-beta-2 :) 

that improve the action selection - by tags and a multi selection
also added some QoL improvements
